sw/device/lib/testing/entropy_testutils.c

```diff
@@ -108,3 +105,19 @@ void entropy_testutils_wait_for_state(const dif_entropy_src_t *entropy_src,
     CHECK_DIF_OK(dif_entropy_src_get_main_fsm_state(entropy_src, &cur_state));
   } while (cur_state != state);
 }
+
+void entropy_testutils_stop_all(void) {
+  const dif_entropy_src_t entropy_src = {
```

Comment:
Just out of curiosity, would this `const` have the same effect of a `static` and prevent the `entropy_src` from being initialized every time this function is executed?